g2p/mappings/langs/kwk/kwk_ipa_to_phonemic_ipa.json

```diff
@@ -0,0 +1,8 @@
+[
+    {"in": "kʲ", "out": "k"},
+    {"in": "xʲ", "out": "x"},
+    {"in": "kʼʲ", "out": "kʼ"},
```

Comment:
I guess these rules can't be simplified by just removing ʲ overall?

Reply:
Possibly but I wasn't confident enough it would be exactly correct, so I preferred not to do it.